Monika After Story/game/script-brbs.rpy

```diff
@@ -471,3 +471,63 @@ label monika_idle_game_callback:
 #        m 1eka "I missed you and I'm glad you're finally back, [player]."
 #        m "I hope I don't have to wait such a long couple of minutes next time, ehehe."
 #    return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_idle_coding",
+            prompt="I'm going to code for a bit",
+            category=['be right back'],
+            pool=True,
+            unlocked=True
+        ),
+        markSeen=True
+    )
+
+label monika_idle_coding:
+    if mas_isMoniNormal(higher=True):
+        m 1eua "Oh! Going to code something?"
+        if persistent._mas_pm_has_code_experience == False:
```

Comment:
I see why you use `== False` here, since using `not persistent._mas_pm_has_code_experience` can be tripped if it's not assigned (`None`). However, for clarity we'll use `is` instead of `==`
```suggestion
        if persistent._mas_pm_has_code_experience is False:
```